docs/diagrams/StorageClassDiagram.puml

```diff
@@ -16,9 +16,12 @@ StorageManager .left.|> Storage
 StorageManager o-right-> UserPrefsStorage
 StorageManager o--> AddressBookStorage
 
+Storage -|> UserPrefsStorage
+Storage -|> AddressBookStorage
```

Comment:
https://github.com/nus-cs2103-AY2021S2/forum/issues/224

Storage Interface only include 5 out of 8 methods from UserPrefsStorage, AddressBookStorage